st2actions/st2actions/runners/remote_command_runner.py

```diff
@@ -61,3 +65,37 @@ def _get_remote_action(self, action_paramaters):
                                   sudo=self._sudo,
                                   timeout=self._timeout,
                                   cwd=self._cwd)
+
+
+class ParamikoRemoteCommandRunner(BaseParallelSSHRunner):
+    def run(self, action_parameters):
+        remote_action = self._get_remote_action(action_parameters)
+
+        LOG.debug('Will execute remote_action: %s.', str(remote_action))
```

Comment:
Now that we are refactoring it, it would be a good time to update this log messages to include context in `extra` :)